sdk/src/main/java/com/mapbox/maps/MapController.kt

```diff
@@ -218,6 +218,7 @@ internal class MapController : MapPluginProviderDelegate, MapControllable {
 
   override fun addWidget(widget: Widget) {
     renderer.renderThread.addWidget(widget)
+    renderer.scheduleRepaint()
   }
 
   override fun removeWidget(widget: Widget) = renderer.renderThread.removeWidget(widget)
```

Comment:
What about other things like `setRotation`, `setTranslation` and `updateBitmap` in the `BitmapWidget`?

Reply:
Good question, but I think after the widget has been added, then updates to any existing widgets should get drawn as part of `MapboxRenderThread.draw`?

Reply:
But true, it does not force a redraw.

Reply:
In the current architecture it is impossible to make a redraw when `BitmapWidget's` `setRotation`, `setTranslation` and `updateBitmap` methods are called directly, since there is no reference to the `MapController`.  Maybe we leave this for another day.